test/runtests.jl

```diff
@@ -212,11 +213,24 @@ function test_ke_dissipation_rate_terms(grid; model_type=NonhydrostaticModel, cl
         @test ε_iso_field isa Field
     end
 
-    ε = KineticEnergyDissipationRate(model; U=0, V=0, W=0)
+    dudz = 2
+    set!(model, u=(x, y, z) -> dudz*z)
+
+    ε = KineticEnergyDissipationRate(model)
+    ε_field = compute!(Field(ε))
+    @test ε isa AbstractOperation
+    @test ε_field isa Field
+
+    true_ε = viscosity(model.closure, model.diffusivity_fields) * dudz^2
+    @test interior(ε_field, 3, 3, 3)[1] == true_ε
+
+    ε = KineticEnergyDissipationRate(model; U=Field(Average(model.velocities.u, dims=(1,2))))
     ε_field = compute!(Field(ε))
     @test ε isa AbstractOperation
     @test ε_field isa Field
 
+    @test interior(ε_field, 3, 3, 3)[1] == 0.0
```

Comment:
I think probably this change will be necessary for tests to pass.

```suggestion
    @test interior(ε_field, 3, 3, 3)[1] ≈ 0.0
```